pkg/ai/types_ai.go

```diff
@@ -0,0 +1,95 @@
+//nolint:revive
+package ai
```

Comment:
Note: These types were added generated, but added manually to facilitate fixing unmarshaling error that was happening due to new entity types being introduced in NerdGraph.